packages/omg-js-childchain/src/index.js

```diff
@@ -147,13 +147,13 @@ class ChildChain {
    * @param {number} toAmount - amount to transact
    * @return {Object} the submitted transaction
    */
-  async sendTransaction (fromAddress, fromUtxos, fromPrivateKeys, toAddress, toAmount) {
+  async sendTransaction (fromAddress, fromUtxos, fromPrivateKeys, toAddress, toAmount, currency) {
```

Comment:
is the `currency` field optional ? if not, will it break current usage of `sendTransaction` ?

Reply:
No, as it is it's not optional.
Hmmm, we _could_ default it to ETH, but I'd kind of prefer to keep it explicit and not give ETH special status.
Besides, I don't think that there _is_ any current usage of `sendTransaction` that this will affect. Before this change `sendTransaction` was bugged and didn't work anyway :)